dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -451,17 +452,17 @@ public Result batchDeleteProcessDefinitionByIds(@ApiIgnore @RequestAttribute(val
                     try {
                         Map<String, Object> deleteResult = processDefinitionService.deleteProcessDefinitionById(loginUser, projectName, processDefinitionId);
                         if(!Status.SUCCESS.equals(deleteResult.get(Constants.STATUS))){
-                            deleteFailedIdList.add(processDefinitionId);
+                            deleteFailedIdList.add(strProcessDefinitionId);
                             logger.error((String)deleteResult.get(Constants.MSG));
                         }
                     } catch (Exception e) {
-                        deleteFailedIdList.add(processDefinitionId);
+                        deleteFailedIdList.add(strProcessDefinitionId);
                     }
                 }
             }
 
             if(!deleteFailedIdList.isEmpty()){
-                putMsg(result, Status.BATCH_DELETE_PROCESS_DEFINE_BY_IDS_ERROR,StringUtils.join(deleteFailedIdList,","));
+                putMsg(result, Status.BATCH_DELETE_PROCESS_DEFINE_BY_IDS_ERROR, Strings.join(deleteFailedIdList,","));
             }else{
```

Comment:
Does `org.apache.parquet.Strings.join` have some advantages than `java.lang.String.join` ?

Reply:
Sorrym, i am very little coding in java, I wrote it in the way of golang ￣□￣｜｜